lib/read.mli

```diff
@@ -135,7 +136,7 @@ val lineseq_from_channel :
 
       See [seq_from_channel] for the meaning of the optional [fin]
       argument.
-      See [from_string] for the meaning of the other optional arguments. *)
+      See [from_string] for the meaning of the other optional arguments and returned exceptions. *)
```

Comment:
"raised exceptions" to be consistent.

Reply:
I initially used "returned" because `lineseq_from_*` doesn't actually raise them but returns with `Exn` instead. I changed it though, because it should be clear from above that all exceptions are via `Exn`.